docs/api-reference/static-analysis.md

```diff
@@ -40,109 +45,219 @@ npm install --save-dev @dbos-inc/eslint-plugin
 ```
 
 Configuring `eslint` can be quite involved, as there are [several complete configuration schemes](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats).
-
-The simplest method is to place an `.eslintrc` file similar to the following in your project directory:
-```json
-{
-  "root": true,
-  "extends": [
-    "plugin:@dbos-inc/dbosRecommendedConfig"
-  ],
-  "plugins": [
-    "@dbos-inc"
-  ],
-  "env": {
-    "node": true,
-    "es6": true
-  },
-  "rules": {
-  },
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "project": "./tsconfig.json"
-  }
-}
-```
+Both of these options require you to set up a `tsconfig.json` file beforehand.
+
+<Tabs groupId="config-types">
+  <TabItem value="flat-config" label="Flat config">
+    <h4> This config style will work with ESLint 8 and above. </h4>
+    Place an `eslint.config.js` file similar to the following in your project directory:
+
+    ```js
+    const { FlatCompat } = require("@eslint/eslintrc");
+    const dbosIncEslintPlugin = require("@dbos-inc/eslint-plugin");
+    const typescriptEslintParser = require("@typescript-eslint/parser");
+    const globals = require("globals");
+    const js = require("@eslint/js");
+
+    const compat = new FlatCompat({
+      baseDirectory: __dirname,
+      recommendedConfig: js.configs.recommended
+    });
+
+    module.exports = [
+      ...compat.extends("plugin:@dbos-inc/dbosRecommendedConfig"),
+
+      { plugins: { "@dbos-inc": dbosIncEslintPlugin } },
+
+      {
+        languageOptions: {
+          parser: typescriptEslintParser,
+          parserOptions: { project: "./tsconfig.json" },
+          globals: { ...globals.node, ...globals.es6 }
+        }
+      },
+
+      { rules: {} }
+    ];
+    ```
+
+    If you'd like stronger typing, you can use `typescript-eslint`'s [`config`](https://typescript-eslint.io/packages/typescript-eslint/#config) helper function.
```

Comment:
Do we use this in the templates?  If not, I'd remove it as extraneous detail.

Reply:
We do - I removed the other, less-well-typed one